### PR TITLE
chore(website): update ecosystem page

### PR DESCRIPTION
**Pull request by dionysuzx**

### CODE REVIEW

File: packages/website/components/EcosystemSection.tsx [hunk 1]
```diff
@@ -130,6 +130,14 @@ const ecosystemData: EcosystemData[] = [
       "User can easily compare and swap multiple chains on top of the Taiko blockchain. A service that links multiple swaps and organizes multiple tokens.",
     filters: [],
   },
+  {
+    icon: "/images/ecosystem/rwbydex.png",
```
Comment (MarcusWentz) [inline packages/website/components/EcosystemSection.tsx:134]: Broken link.
Should be
```
icon: "/images/ecosystem/rwbydex.png",
```

**Review by MarcusWentz — CHANGES_REQUESTED** [review]

**Comment by MarcusWentz** [comment]
render

**Review by dantaik — DISMISSED** [review]

**Comment by 2manslkh** [comment]
I don't see how Taiko was integrated

**Review by davidtaikocha — DISMISSED** [review]

**Comment by d1onys1us** [comment]
> I don't see how Taiko was integrated

updated, the link was wrong for rubydex. also this PR has the right linke for owlto.finance

**Review by davidtaikocha — APPROVED** [review]

**Review by MarcusWentz — APPROVED** [review]

**Comment by 2manslkh** [comment]
Closes #14173

**Pull request merged**